man/systemd.netdev.xml

```diff
@@ -911,6 +911,13 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>6rdPrefix=</varname></term>
+        <listitem>
+          <para>The <literal>6rdPrefix=</literal> parameter configures the tunnel as a 6rd tunnel, with the given
+          ISP-specific IPv6 prefix.</para>
```

Comment:
Please specify what kind value can be specified here. I guess an IPv6 address with non-zero prefix length.